src/types/date-time-picker.ts

```diff
@@ -1,20 +1,11 @@
 import type { AnyElementOf } from '@/types/utils'
-
+import type { TimePeriod } from '@kong-ui-public/analytics-utilities'
 export interface TimeRange {
```

Comment:
Can import this too from analytics-utilities

https://github.com/Kong/public-ui-components/blob/8c720c694daf40b45947886357d1b44bb3b4647e/packages/analytics/analytics-utilities/src/types/datepicker-selection.ts

Reply:
No, kongponents cannot have a dependency on another kong package